src/plugins/postcss-icss-parser.js

```diff
@@ -56,41 +56,45 @@ export default postcss.plugin(
           }
 
           const token = node.value;
-          const importIndex = imports[`$${token}`];
+          const replacement = importReplacements[token];
 
-          if (typeof importIndex === 'number') {
+          if (replacement != null) {
```

Comment:
Why not `if (replacement) {}`?

Reply:
it could be! The old want needed to check for `0` but that's not the case anymore so lets do that!

Reply:
:+1: